apps/emqx_rule_engine/src/emqx_rule_engine_api.erl

```diff
@@ -661,3 +697,14 @@ run_fuzzy_match(E = {_Id, #{from := Topics}}, [{from, like, Pattern} | Fuzzy]) -
         run_fuzzy_match(E, Fuzzy);
 run_fuzzy_match(E, [_ | Fuzzy]) ->
     run_fuzzy_match(E, Fuzzy).
+
+rule_engine_update(Params) ->
+    case emqx_rule_api_schema:check_params(Params, rule_engine) of
```

Comment:
For my own information: would setting `check_schema => true` in [l. 134](https://github.com/emqx/emqx/pull/10336/files#diff-650d13fe1363c91b6a7c19323b55af7bc8d9854504df27afd2c9a912cad06c32R134) obviate the need for this manual check?